package.json

```diff
@@ -27,6 +29,7 @@
     "eslint-plugin-prettier": "^3.1.3",
     "husky": "^4.2.5",
     "jest": "^25.3.0",
+    "nock": "12.0.3",
```

Comment:
I know we're going to bake recording into the SDK at some point but until then you should use pollyjs, it's what the SDK will be using and what we use in other integrations.

Reply:
Replaced `nock` with `polly` in #2